added new f1_ackermann_harmonic car and simple_circuit_harmonic.sdf world in ackermann_cars

nikhilg2603 opened the pull request:
This is with reference to issue https://github.com/JdeRobot/RoboticsInfrastructure/issues/518. 3 files have been added, f1_ackermann_harmonic amd simple_circuit_harmonic package under the directory /CustomRobots/ackermann_cars/model. and simple_circuit_harmonic.sdf under the directory /CustomRobots/ackermann_cars/worlds.

nikhilg2603 committed:
added new f1_ackermann_harmonic car and simple_circuit_harmonic.sdf world in ackermann_cars

nikhilg2603 committed:
corrected the sizing issue for ackermann car

javizqh commented:
Please add a launcher and a database entry for the world

pariaspe commented on CustomRobots/ackermann_cars/models/f1_renault_ackermann_harmonic/model.sdf:
```diff
@@ -0,0 +1,546 @@
+        <geometry>
+          <mesh>
+            <scale>0.001 0.001 0.001</scale>
+            <uri>f1_renault_ackermann_harmonic/Renault/fr_1.stl</uri>
```
add missing `model://` uri clause, can't edit it myself since your PR don't admit changes

pariaspe reviewed (changes requested):
Please take a look at the requested change.

pariaspe commented:
> Please add a launcher and a database entry for the world

Hi @javizqh , Nikhil as part of his gsoc project is only in charge of migrating models and worlds (and not full exercise migration). Can we skip launcher and database entry creation (at least in this PR)?

javizqh commented:
The database entry can be skipped, but the Launcher is needed to test that the car loads correctly. If you can test it with other methods it is fine without both.

nikhilg2603 committed:
Update ackermann model.sdf

Resolving the reviews, corrected the model.sdf in f1_renault_ackermann_harmonic

pariaspe commented:
> The database entry can be skipped, but the Launcher is needed to test that the car loads correctly. If you can test it with other methods it is fine without both.

Tested locally and ackermann model is working

pariaspe committed:
Merge branch 'humble-devel' into humble-devel
